packages/block-library/src/index.js

```diff
@@ -227,6 +226,24 @@ const getAllBlocks = () => {
 		queryTitle,
 		postAuthorBiography,
 	];
+
+	// Only add the classic block in WP Context
+	// and when either of the following is true:
+	// - TinyMCE experiment is disabled;
+	// - there is a classic block in the current post's content;
+	// - the cookie to require TinyMCE is set.
```

Comment:
Can we unify "classic block in current post's content" and "requireTinyMCE" is set? Can we just set the cooking when the current post has classic block for instance?

Reply:
We're no longer working with a cookie, we're rather working with a query argument since https://github.com/WordPress/gutenberg/pull/50387/commits/896f8cf5a8b993dd409469af249a70fdc97c0467, so I think this should no longer be relevant - let me know what you think.